test/protocol/ProtocolDiamondTest.js

```diff
@@ -970,7 +970,7 @@ describe("ProtocolDiamond", async function () {
         ).to.revertedWith(RevertReasons.CONTRACT_NOT_ALLOWED);
       });
 
-      it("should revert with library reason if not supplied by implementation", async () => {
+      it("Should revert with library reason if not supplied by implementation", async () => {
```

Comment:
"Library reason if not supplied by implementation"

Reply:
done